css/app.css

```diff
@@ -117,6 +117,7 @@ img {
   -webkit-box-orient: horizontal;
   -webkit-box-pack: justify;
   display: -webkit-flex;
```

Comment:
Can this just be `display: flex;`?

Reply:
It sure can. This patch is just about centering though. The next patch I'm working on which replaces the close file icon with MD has an overhaul of the layout and use flex box.